index.html

```diff
@@ -13,21 +13,21 @@
 </head>
 
 <body>
-  <div id="header">
-    <header id="tool-bar">
-      <div id="mylogo">
-        <a href="#heading"><img src="img/MyLogo.png" alt="Logo"></a>
-      </div>
-      <nav id="nav">
-        <div id="btn-menu"><img src="img/Normal-Button/Tertiary/Icons/Icon-Menu.png" alt=""></div>
-        <ul id="menu-lst">
-          <li><a href="#portfolio" class="menu-lst-txt">Portfolio</a></li>
-          <li><a class="menu-lst-txt" href="#About-my-self">About</a></li>
-          <li><a class="menu-lst-txt" href="#Contact-Section">Contact</a></li>
-        </ul>
-      </nav>
-    </header>
-    <section id="main-head-line">
+  <header id="tool-bar">
+    <div id="mylogo">
+      <a href="#heading"><img src="img/MyLogo.png" alt="Logo"></a>
+    </div>
+    <nav id="nav">
+      <div id="btn-menu"><img src="img/Normal-Button/Tertiary/Icons/Icon-Menu.png" alt=""></div>
+      <ul id="menu-lst">
+        <li><a href="#portfolio" class="menu-lst-txt">Portfolio</a></li>
+        <li><a class="menu-lst-txt" href="#About-my-self">About</a></li>
+        <li><a class="menu-lst-txt" href="#Contact-Section">Contact</a></li>
+      </ul>
```

Comment:
- [ ] Kindly adjust the spacing between the hamburger menu and the logo to match the given design.
![image](https://user-images.githubusercontent.com/62290545/190221092-06ef88da-ba07-4ab3-9a20-74c9bc264708.png)